workspaces/marketplace/plugins/marketplace/src/components/MarketplacePluginInstallContent.tsx

```diff
@@ -69,25 +148,179 @@ export const MarketplacePluginInstallContent = ({
     }
   }, [codeEditor, copyToClipboard]);
 
+  const navigate = useNavigate();
+  const theme = useTheme();
+  const borderColor = theme.palette.mode === 'dark' ? '#a3a3a3' : '#c7c7c7';
+  const installationInstructions = plugin.spec?.installation;
+  const examples = packages[0]?.spec?.appConfigExamples;
+  const showRightCard = installationInstructions || examples;
+  // const [yamlText, setYamlText] = useState('');
+  const [tabIndex, setTabIndex] = useState(0);
+
+  const handleTabChange = (_: any, newValue: React.SetStateAction<number>) => {
+    setTabIndex(newValue);
+  };
+
   return (
-    <div style={{ display: 'flex', flexDirection: 'column', height: '100%' }}>
-      <div style={{ flex: 1 }}>
-        <CodeEditor defaultLanguage="yaml" onLoaded={onLoaded} />
-      </div>
-      <div style={{ paddingTop: 16 }}>
+    <Box
+      sx={{
+        height: '80vh',
```

Comment:
Can we move the checkboxes to the buttons and make the editor / sidebar content "flex: 1" and the checkboxes and buttons pick up the space they have.

Reply:
I did the change you suggested, but we still need a height value so that the "flex: 1" can work properly.